src/Flame/Renderer/Renderer.js

```diff
@@ -41,7 +40,51 @@ function runEvent(handler) {
 
 exports.h_ = h;
 
-exports.thunk_ = thunk;
+function copyToThunk(vnode, thunk) {
+    thunk.elm = vnode.elm;
```

Comment:
Duplicate assignment (same as line 50)

Reply:
Oddly, this is how snabbdom has it. Guess they didn't have a review :S